nsqd/http.go

```diff
@@ -226,6 +231,32 @@ func (s *httpServer) getTopicFromQuery(req *http.Request) (url.Values, *Topic, e
 	return reqParams, s.ctx.nsqd.GetTopic(topicName), nil
 }
 
+func (s *httpServer) doLookup(req *http.Request) (interface{}, error) {
```

Comment:
> I'm not sure how much of the lookupd endpoints you wanted to land here vs follow on PR's; Some translate easily, and some have some things i hadn't thought about (tombstoning, pre-creating in registry db, deleting, etc) for making this fully back nsqadmin.

Let's defer this conversation to a future PR because there are other alternatives.  For one, I have #323 inching closer to completion.  And this work here paves the way for nsqadmin to *participate* in the gossip cluster as a means of having data pushed to it + querying :flushed:.